lib/ios-deploy.js

```diff
@@ -0,0 +1,73 @@
+import { exec } from 'teen_process';
+import { fs } from 'appium-support';
+import logger from './logger';
+import { retryInterval } from 'asyncbox';
+
+const IOSDEPLOY_PATH = `ios-deploy`;
+
+class IOSDeploy {
```

Comment:
I'm wondering if this should extend from the basedriver

Reply:
Unless Im not understanding what this is for

Reply:
This is meant to interface with ios-deploy (https://github.com/ios-control/ios-deploy)
It's pretty much copied from appium-xcuitest-driver (https://github.com/appium/appium-xcuitest-driver/blob/master/lib/ios-deploy.js)